modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/dump/Dump.java

```diff
@@ -247,7 +257,11 @@ public DumpedPartitionIterator iterator(String node, int group, int part) {
         FileIO dumpFile;
 
         try {
-            dumpFile = ioFactory.create(new File(dumpGroupDirectory(node, group), PART_FILE_PREFIX + part + DUMP_FILE_EXT));
+            File file = new File(dumpGroupDirectory(node, group), PART_FILE_PREFIX + part + DUMP_FILE_EXT);
```

Comment:
We have acceess to dump meta here.
So we know exactly what file extension we are expecting to find.